backstop/tests/file-upload.html

```diff
@@ -736,14 +694,13 @@ <h2>Not wrapping anything</h2>
           />
           <!-- Use upload.svg from generic-icons package. -->
           <svg
-            class="iui-icon"
             viewBox="0 0 16 16"
             aria-hidden="true"
           >
             <path d="m2 5h4v7h4v-7h4l-6-5zm12 6v3h-12v-3h-2v5h16v-5z" />
           </svg>
           <label for="file">
-            <strong>Choose a file</strong><br />or drag & drop it here.
+            Choose a file<br />or drag & drop it here.
```

Comment:
I'm still thinking. Maybe we should use `white-space: pre-line` so that user can pass a single localized string separated by newline (`\n`).

Reply:
Separating into two elements after discussion because we don't want the whole thing to be clickable